### PR TITLE
#3 Create Checkout Flow

### PR DESCRIPTION
**Pull request by kagaba-etienne**
#### What does this PR do?
- Creates and order after checkout
- Calculates discount from applied coupons
#### Description of Task to be completed

> The backend needs to handle the checkout process, including validating and processing the order, updating inventory, and updating the customer's account.

#### How should this be manually tested?
- Checkout in branch `ft-checktout-3` and pull most recent changes
- Start the server by running `npm run start`
- Make sure you have a user account already, and few products in your cart before proceeding checkout.
- Make a POST request to this route `/api/checkout` with a requestBody containing the following properties `country, street_address, town, state, post_code, email` AKA billing information. Wait for a successful response. Kudos !!! You have just initiated an order.

#### Relevant pivotal tracker stories?
- [Card #3](https://trello.com/c/LUfLttI6)

### CODE REVIEW

**Review by bryan936M — APPROVED** [review]
Nice!

**Review by ALVINdimpos — APPROVED** [review]

**Review by vanessa-abewe — APPROVED** [review]

**Review by yobuir — APPROVED** [review]
nice work #resolve_conflicts

**Review by NTElissa — APPROVED** [review]

**Review by NTElissa — COMMENTED** [review]
Good progress

File: src/database/models/index.js [hunk 1]
```diff
@@ -13,14 +13,27 @@ const env = _env.NODE_ENV || 'development';
...
 const config = Config[env];
 const db = {};
-let sequelize;
+// let sequelize;
 
 
 
-if (config.use_env_variable) {
-  sequelize = new Sequelize(_env[config.use_env_variable], config);
+// if (config.use_env_variable) {
```
Comment (chanceDira) [inline src/database/models/index.js]: Remove these comments

File: tests/Product.test.js [hunk 2]
```diff
@@ -83,4 +88,19 @@ describe('Product API', function() {
         });
     });
   });
+  // describe('DELETE api/product/:id', function() {
```
Comment (chanceDira) [inline tests/Product.test.js]: Remove these comments too

**Review by chanceDira — CHANGES_REQUESTED** [review]
Good job @kagaba-etienne just one thing

**Review by chanceDira — APPROVED** [review]
Good job @kagaba-etienne

**Pull request merged**